languages/csharp/exercises/concept/calculator-conundrum/.meta/Example.cs

```diff
@@ -1,35 +1,32 @@
 using System;
-static class SimpleCalculator
+
+public static class CalculatorConundrum
```

Comment:
@mikedamay This might be a case where the original name is better the exercise-based name. What do you think?

Reply:
changing `public static class CalculatorConundrum` to `public static class SimpleCalculator`